app/src/main/java/com/example/tasky/agenda/presentation/itemdetail/viewmodel/EventDetailViewModel.kt

```diff
@@ -258,6 +258,17 @@ class EventDetailViewModel @Inject constructor(
             EventDetailEvent.PhotoClickedResolved -> {
                 state = state.copy(navigatePhotoViewer = null)
             }
+
+            is EventDetailEvent.OnAttendeeDelete -> {
+                val attendees = state.event.attendees.toMutableList()
+                val attendeeToDelete = attendees.first { it.userId == event.attendee.userId }
+                attendees.remove(attendeeToDelete)
```

Comment:
Pretty sure you could

state = state.copy(event.... attendees = attendees - attendeeToDelete)

And the val attendees could be the state.event.attendees. no need for it to be mutable

I'd need to test it (not home rn), but it should work